### PR TITLE
915 configurable encounter search

### LINKED ISSUE #915: adapt encountersearch.jsp to be configurable
## Feature description and context
We have a stable react search page, but the indexing on large scale platforms is not up to par for large scale platforms. we want to have a configuration option for large scale platforms to make use of the old search page as we iterate through improvements on opensearch

## Feature sign-off requirements
- new menu option for "Encounter Search (Classic)"
- configurable **Follow implementation strategy from submit.jsp**

### PR DESCRIPTION
**Pull request by erinz2020**
1. update commonConguration.properties to add a toggle to show/hide encounterSearch.jsp
2. reflect this in jsp, react and servlet template headers
3. new menu option is `Encounter Search (Classic)`

PR fixes #915

### CODE REVIEW

File: src/main/resources/bundles/commonConfiguration.properties
Comment (TanyaStere42) [inline src/main/resources/bundles/commonConfiguration.properties:1]: in commonConfig, can you address the typo in `showClassicEncouners`? Encounters is missing the `t`

**Review by TanyaStere42 — CHANGES_REQUESTED** [review]
looks like this is missing the changes for the servletResponseTemplate.htm?

File: src/main/resources/bundles/commonConfiguration.properties
Comment (erinz2020) [reply src/main/resources/bundles/commonConfiguration.properties:1]: oops, this typo took me 10 mins to find out why showClassicEncounters is getting undefined, and now I forgot to push this commit! thank you so much for pointing out!

**Review by naknomum — APPROVED** [review]
other than the typo, i am signing off on this because the java looks sound.

**Comment by erinz2020** [comment]
> looks like this is missing the changes for the servletResponseTemplate.htm?

added. weird, I definitely updated this too earlier today

**Pull request merged**